caom2utils/caom2utils/tests/test_collections.py

```diff
@@ -101,7 +101,8 @@ def test_differences(directory):
     assert len(expected_fname) == 1
     expected = _read_observation(expected_fname[0])  # expected observation
     assert len(expected.planes) == 1
-    prod_id = [p.product_id for p in expected.planes.values()][0]
+    plane_uri = [p.uri for p in expected.planes.values()][0]
+    prod_id = plane_uri.split('/')[-1]
```

Comment:
It should probably be added to a list somewhere to eventually change the `productID` parameter to the `planeURI` parameter, but the when of doing that is not urgent. Same comment for the `help` for the `observation` parameter.

Reply:
Technical debt.